pom.xml

```diff
@@ -195,6 +195,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+            	<groupId>org.apache.maven.plugins</groupId>
+            	<artifactId>maven-invoker-plugin</artifactId>
+            	<configuration>
+                        <mergeUserSettings>true</mergeUserSettings>
```

Comment:
why so many indentation spaces? do you see how may do we use in the rest of the document?